lib/package.json

```diff
@@ -13,6 +13,7 @@
   },
   "dependencies": {
     "aws-sdk": "*",
+    "change-case": "^4.1.2",
```

Comment:
Think this can be removed?
(See no usage in the module itself, guess this is something left from testing with sammy?)

Reply:
Oh.. It looks like I added it by mistake. I will remove it tomorrow. Or you also can do it.

Reply:
No problem, resolved it 👍